specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/virtualWan.json

```diff
@@ -2634,6 +2634,231 @@
+          "default": {
+            "description": "The operation set shared key of VpnLink connection could not be completed.",
+            "schema": {
+              "$ref": "./networkWatcher.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ResetVpnLinkConnection": {
+            "$ref": "./examples/VpnSiteLinkConnectionSharedKeySet.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnGateways/{gatewayName}/vpnConnections/{connectionName}/vpnLinkConnections/{linkConnectionName}/sharedKey/reset": {
+      "post": {
+        "tags": [
+          "vpnLinkConnections"
+        ],
+        "operationId": "VpnLinkConnections_Reset_SharedKey",
+        "description": "Put the shared key of VpnLink connection specified.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "gatewayName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the gateway."
+          },
+          {
+            "name": "connectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the vpn connection."
+          },
+          {
+            "name": "linkConnectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the vpn link connection."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConnectionResetSharedKey"
+            },
+            "description": "Parameters supplied to the begin reset VpnLink connection shared key operation through network resource provider."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation Reset shared key of VpnLink connection.",
+            "schema": {
+              "$ref": "#/definitions/ConnectionResetSharedKey"
+            }
+          },
+          "202": {
+            "description": "Accepted and the operation ResetConnection will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/ConnectionResetSharedKey"
```

Comment:
[ARMBlockingComment]
The 202 response can't have schema.

Reply:
changed it to 201

Reply:
See above comment. 201 isn't a valid POST status code for ARM.